nav2_costmap_2d/plugins/costmap_filters/speed_filter.cpp

```diff
@@ -90,6 +90,7 @@ void SpeedFilter::initializeFilter(
   // Reset speed conversion states
   base_ = BASE_DEFAULT;
   multiplier_ = MULTIPLIER_DEFAULT;
+  percentage_ = true;
```

Comment:
Lets default to absolute, e.g. `false`

Reply:
Done in https://github.com/ros-planning/navigation2/pull/2149/commits/6bd018bee773e8ae042062cde5963646a93cba25#diff-d344e376ecbc8e5ecd5045731e13855c6d2a9a955cdaee9b31dddbbee7681466L93